docs/cmdline-opts/ipfs-gateway.d

```diff
@@ -0,0 +1,43 @@
+c: Copyright (C) 2023, Mark Gaiser, <markg85@gmail.com>
+SPDX-License-Identifier: curl
+Long: ipfs-gateway
+Arg: <URL>
+Help: Defines the gateway to handle IPFS protocol requests.
+Added: 8.3.0
```

Comment:
You can probably bump this top `8.4.0` now, as it looks like it should merge there.